IIPS/iip-6.md

```diff
@@ -143,6 +143,240 @@ Returns the hash for a given address's reverse records.
 def node(addr: Address) -> str : 
 ```
 
+## Implementation
+
+[Draft Version]  
+The ICON Name Service has impelement as below.
+
+### INS Registry Implementation
+
+The INS Registry is a root storage of the ICON Name Service.
+
+```python
+class INSRegistry(IconScoreBase):
+
+    __OWNER = 'owner'
+    __RESOLVER = 'resolver'
+    __TTL = 'ttl'
+
+    @eventlog(indexed=2)
+    def Transfer(self, node: bytes, owner: Address) : pass
+
+    @eventlog(indexed=3)
+    def NewOwner(self, node: bytes, label: bytes, owner: Address) : pass
+
+    @eventlog(indexed=2)
+    def NewResolver(self, node: bytes, resolver: Address) : pass
+
+    @eventlog(indexed=2)
+    def NewTTL(self, node: bytes, ttl: int) : pass
+
+    def __init__(self, db: IconScoreDatabase, addr_owner: Address) -> None:
```

Comment:
`addr_owner` has been removed from `__init__` function definition.  Please remove the param.